src/app/shared/helpers/domain.ts

```diff
@@ -0,0 +1,6 @@
+export function getDomain(href: string): string {
```

Comment:
should it be called "getRootDomain"? From my point of view, the domain of `https://sub.example.com/api/stuff ` is `sub.example.com`, not `example.com`

Reply:
Fixed. It became dead code.